tsp-core/src/main/java/pl/edu/agh/iet/tsp/core/routing/json/PagedResult.java

```diff
@@ -0,0 +1,16 @@
+package pl.edu.agh.iet.tsp.core.routing.json;
+
+import java.util.List;
+
+/**
+ * @author Wojciech Pachuta.
+ */
+public class PagedResult<T> {
+    public final List<T> data;
+    public final String next;
```

Comment:
dokladnie. z tego co patrzylem w internijetach to cos takiego jest w miare standardowa praktyka